modules/cache/src/main/java/org/eclipse/xpanse/modules/cache/CaffeineCacheConfig.java

```diff
@@ -56,4 +69,32 @@ private Cache<Object, Object> getServiceFlavorPriceCache() {
                 : DEFAULT_CACHE_EXPIRE_TIME_IN_MINUTES;
         return Caffeine.newBuilder().expireAfterWrite(duration, TimeUnit.MINUTES).build();
     }
+
+
+    private Cache<Object, Object> getCredentialsCache() {
+        Cache cache = Caffeine.newBuilder()
+                .expireAfter(new CredentialCaffeineCacheExpiry())
+                .removalListener((CredentialCacheKey key,
+                                  AbstractCredentialInfo credential,
+                                  RemovalCause cause) -> {
+                    if (Objects.nonNull(key)) {
+                        log.info("Credential cache removed, key: {}, cause: {}", key, cause);
+                    }
+                })
+                .build();
+        return cache;
+    }
+
+    private Cache<Object, Object> getMonitorMetricsCache() {
+        Cache cache = Caffeine.newBuilder()
+                .expireAfterWrite(10, TimeUnit.MINUTES)
```

Comment:
this expiry does not match the value in the below code. 
I think it is better to move this as a config value in properties file. 
infact, would be better to have this configuration in properties file.. similar to other caches 

https://github.com/eclipse-xpanse/xpanse/blob/abc2b1c8b62b13cdee5103ac230153818039b882/modules/cache/src/main/java/org/eclipse/xpanse/modules/cache/monitor/MonitorMetricsStore.java#L92

Reply:
Updated as suggestion.